### ISSUE TITLE
Can I use this on videos?

### ISSUE THREAD

**Issue by Jeredriq** [IDEA-Research/Grounded-Segment-Anything#477]

**Comment by rentainhe** [comment]
Sure, you can use Grounded-SAM with Tracking model for video data

**Comment by Jeredriq** [comment]
@rentainhe thanks! Is there a google colab or huggingface page for this?

**Comment by rentainhe** [comment]
> @rentainhe thanks! Is there a google colab or huggingface page for this?

Yes, you can refer to https://github.com/hkchengrex/Tracking-Anything-with-DEVA for more details
